lib/docker/util.rb

```diff
@@ -141,15 +141,16 @@ def create_dir_tar(directory)
 
   def create_relative_dir_tar(directory, output)
     Gem::Package::TarWriter.new(output) do |tar|
-      Find.find(directory) do |prefixed_file_name|
-        stat = File.stat(prefixed_file_name)
-        next unless stat.file?
-
-        unprefixed_file_name = prefixed_file_name[directory.length..-1]
-        add_file_to_tar(
-          tar, unprefixed_file_name, stat.mode, stat.size, stat.mtime
-        ) do |tar_file|
-          IO.copy_stream(File.open(prefixed_file_name, 'rb'), tar_file)
+      Dir.chdir(directory) do
```

Comment:
`Dir.chdir` isn't threadsafe actually, which is why I went with the slicing approach.